tbx/core/tests/test_models.py

```diff
@@ -0,0 +1,63 @@
+from tbx.core.factories import HomePageFactory, StandardPageFactory
+from tbx.core.models import HomePage, StandardPage
+from wagtail.models import Site
+from wagtail.test.utils import WagtailPageTestCase
+from wagtail.test.utils.form_data import (
+    nested_form_data,
+    rich_text,
+    streamfield,
+)
+
+
+class TestHomePageFactory(WagtailPageTestCase):
+    def test_create(self):
+        HomePageFactory()
+
+
+class TestStandardPageFactory(WagtailPageTestCase):
+    def test_create(self):
+        StandardPageFactory()
+
+
+class TestStandardPage(WagtailPageTestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        site = Site.objects.get(is_default_site=True)
+        root = site.root_page.specific
+        cls.home_page = HomePageFactory(parent=root)
+
+        site.root_page = cls.home_page
+        site.save()
+
+    def test_can_create_standard_page_under_home_page(self):
+        self.assertCanCreateAt(HomePage, StandardPage)
+
+    def test_can_create_standard_page_with_supplied_post_data(self):
+        self.login()
+        self.assertCanCreate(
+            self.home_page,
+            StandardPage,
+            nested_form_data(
+                {
+                    "title": "Standard Page",
+                    "body": streamfield(
+                        [
+                            ("h2", "Here's a level 2 heading"),
+                            (
+                                "paragraph",
+                                rich_text("<p>Here's a <em>short</em> paragraph</p>"),
+                            ),
+                        ]
+                    ),
+                }
+            ),
+        )
```

Comment:
Just keep in mind that tests are code too, so the following still apply:
- It should have 'necessity': The benefit of its existence should be clear/obvious
- The cost of long-term maintenance needs to be factored in

I feel a little like we're falling into the trap here of 'adding tests that look like they are doing something useful' - but in reality, aren't really providing benefit. The standard page is just a Page model using a few standard model fields in a documented way, and isn't making any overrides to edit form/save behaviour - so what we are really testing here is "Does Wagtail still work in the documented way" (which is up to Wagtail to test)